examples/axes_grid1/demo_axes_rgb.py

```diff
@@ -81,7 +79,8 @@ def demo_rgb2():
     return ax
 
 
-demo_rgb()
-demo_rgb2()
+with plt.rc_context({"image.origin": "lower"}):
```

Comment:
I believe the image origin belongs to the specific plot and should be defined there, not here.

Reply:
actually the example works just fine with origin="upper" so I completely got rid of setting the origin.

Reply:
👍 That's actually the better way. Usually the pixel (0, 0) is at the top left with images.

Original: https://matplotlib.org/devdocs/gallery/axes_grid1/demo_axes_rgb.html
New: https://9680-7439715-gh.circle-artifacts.com/0/home/circleci/project/doc/build/html/gallery/axes_grid1/demo_axes_rgb.html